### ISSUE TITLE
Is this a bug or not? 15000 hours for training XL model

### ISSUE THREAD

**Issue by Samael-1976** [bmaltais/kohya_ss#1534]
Is this normal or I've some kind of bug?
Almost 15000 hours (over 600 days) for train a model in XL with a 2060 12gb of ram.
236 images
300 repeats
70000 steps in total
Learning rate 4e-7

![train](https://github.com/bmaltais/kohya_ss/assets/128039181/97c5e949-c96d-4ead-9962-11da5d26dc6a)

Can I change something in my parameters for go faster?

**Comment by DevArqSangoi** [comment]
Not a bug, your parameters are using more VRAM than your GPU has, so the training script starts using regular RAM as well, which makes the training incredibly slow. Review your parameters, optimizer, etc.

**Comment by Samael-1976** [comment]
> Not a bug, your parameters are using more VRAM than your GPU has, so the training script starts using regular RAM as well, which makes the training incredibly slow. Review your parameters, optimizer, etc.

![01](https://github.com/bmaltais/kohya_ss/assets/128039181/e90b68f2-e20c-4083-ad6a-90f2fd71d780)
![02](https://github.com/bmaltais/kohya_ss/assets/128039181/e6c3bc49-4e64-490c-9c7a-5d3360b33a99)
![03](https://github.com/bmaltais/kohya_ss/assets/128039181/a4ad7052-7baf-44a8-9dbf-4bb316829bf6)
![03a](https://github.com/bmaltais/kohya_ss/assets/128039181/26c59e39-c06d-43a0-b94f-0e6609e1b4d4)
![04](https://github.com/bmaltais/kohya_ss/assets/128039181/1143cf8f-b7f4-42f6-965c-8c4823053e10)
![05](https://github.com/bmaltais/kohya_ss/assets/128039181/4ce13ed0-4263-4da9-ac1b-fbf4d672abe3)
![06](https://github.com/bmaltais/kohya_ss/assets/128039181/2b8e8e1f-bd73-48d3-8921-6324f431d465)

**Comment by DevArqSangoi** [comment]
Don't use finetune; it's what consumes the most VRAM. Train a regular Lora or a Lycoris instead.

**Comment by SilverRider76** [comment]
> Not a bug, your parameters are using more VRAM than your GPU has, so the training script starts using regular RAM as well, which makes the training incredibly slow. Review your parameters, optimizer, etc.

But I don't want do a Lora... I want do a full checkpoint.

**Comment by SilverRider76** [comment]
> Try Dreambooth then, using captions and without reg images. If it still doesn't work, I'm sorry, but to create an SDXL ckpt without taking 15,000 hours, you'll need more VRAM.

Thank you, I will try!
